config/scheduler-chromeos.yaml

```diff
@@ -50,60 +50,106 @@ _anchors:
       type: lava
       name: lava-collabora
 
-  test-job-mediatek: &test-job-mediatek
+  test-job-arm64-mediatek: &test-job-arm64-mediatek
     <<: *lava-job-collabora
     event:
       channel: node
-      name: kbuild-gcc-10-arm64-chromeos-mediatek
+      name: kbuild-gcc-10-arm64
       result: pass
     platforms: *mediatek-platforms
 
-  test-job-pineview: &test-job-pineview
+  test-job-arm64-qualcomm: &test-job-arm64-qualcomm
+    <<: *lava-job-collabora
+    event:
+      channel: node
+      name: kbuild-gcc-10-arm64
+      result: pass
+    platforms: *qualcomm-platforms
+
+  test-job-chromeos-amd: &test-job-chromeos-amd
+    <<: *lava-job-collabora
+    event:
+      channel: node
+      name: kbuild-gcc-10-x86-chromeos-amd
+      result: pass
+    platforms: *amd-platforms
+
+  test-job-chromeos-intel: &test-job-chromeos-intel
+    <<: *lava-job-collabora
+    event:
+      channel: node
+      name: kbuild-gcc-10-x86-chromeos-intel
+      result: pass
+    platforms: *intel-platforms
+
+  test-job-chromeos-mediatek: &test-job-chromeos-mediatek
     <<: *lava-job-collabora
     event:
       channel: node
-      name: kbuild-gcc-10-x86-chromeos-pineview
+      name: kbuild-gcc-10-arm64-chromeos-mediatek
       result: pass
-    platforms: *intel-pineview-platforms
+    platforms: *mediatek-platforms
 
-  test-job-qualcomm: &test-job-qualcomm
+
+  test-job-chromeos-qualcomm: &test-job-chromeos-qualcomm
     <<: *lava-job-collabora
     event:
       channel: node
       name: kbuild-gcc-10-arm64-chromeos-qualcomm
       result: pass
     platforms: *qualcomm-platforms
 
-  test-job-stoneyridge: &test-job-stoneyridge
+
+  test-job-x86-amd: &test-job-x86-amd
     <<: *lava-job-collabora
     event:
       channel: node
-      name: kbuild-gcc-10-x86-chromeos-stoneyridge
+      name: kbuild-gcc-10-x86
       result: pass
```

Comment:
(optional) You could reuse `test-job-chromeos-amd` here and only change the event:

```yaml
    <<: *test-job-chromeos-amd
    event:
      channel: node
      name: kbuild-gcc-10-x86
      result: pass
```